Blocks/src/Controller/Admin/BlocksController.php

```diff
@@ -77,7 +71,7 @@ public function afterConstruct() {
  */
 	public function beforeFilter(Event $event) {
 		parent::beforeFilter($event);
-		$this->Security->unlockedActions[] = 'admin_toggle';
+		$this->Security->config('unlockedActions', 'admin_toggle');
```

Comment:
Maybe you're right.

I changed this only because I was getting `unlockedActions` not defined error or similar.